src/main/java/pl/sda/javagda28/tourmanagingcrud/entity/Event.java

```diff
@@ -3,8 +3,19 @@
 import lombok.AllArgsConstructor;
 import lombok.Data;
 import lombok.NoArgsConstructor;
+import lombok.ToString;
 
-import javax.persistence.*;
```

Comment:
tu dobrze usunąłeś wildcard. gadaliśmy pewnie o tym kiedyś i masz to w konfiguracji, tylko nie wykonałeś w tamtych klasach optimize imports pewnie :D